contributors/devel/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
Unfortunately no.  There's a distinction between the ContribX work and design proposals work.  Ideally, the design proposals will get organized into sig subdirs and owners can be tied to sig aliases (that's for a different PR).